.github/workflows/on-tag-push-create-release.yml

```
@@ -0,0 +1,269 @@
+          core.setOutput('commit-sha', latestRun.head_sha);
+
+  download-artifacts:
+    name: Download Latest Build Artifacts
+    needs: common-vars
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACTS_DIR: ./artifacts
+      RELEASE_DIR: ./release
+
+    steps:
+    - name: Create artifact directories
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)